Tests/JsonApiSerializerTests.cs

```diff
@@ -68,6 +68,22 @@ public void ConditionallyAppliesFilters()
 
         }
 
+		[Fact(DisplayName = "Uses query fieldset expressions if specified")]
```

Comment:
I think we need a test where we ask for a field that's not on the model (should get nothing in that case).

Reply:
Added the test in a0ee8584e64fe2562d16f05a175a6225962dee00. I think only the attributes should be empty as id and type field are still returned which I think would be correct behaviour at this point.